lib/management/configuration.js

```diff
@@ -171,7 +171,7 @@ function patchConfiguration(newConf, log, cb) {
                 _config.setLocationConstraints(locations);
             } catch (error) {
                 log.info('could not apply configuration version location ' +
-                    'constraints', { error });
+                    'constraints', { msg: error.message, trace: error.stack });
```

Comment:
Two comments:

* We mostly use `error` attribute to log the error message instead of `msg`

* I don't recommend logging the stack trace except for `log.trace` kind of logs, in my experience they bloat the logs and make them less readable while usually not being this useful to track down the issues (one may disagree of course)